ci/set_numpy_pandas_integration.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -xeo pipefail
+
+pip-compile ./ci/numpy_pandas_requirements.txt -o ./ci/numpy_pandas_req_pinned.txt
```

Comment:
`ci/numpy_pandas_requirements.txt` isn't that file missing in this PR?

Reply:
It's there.

Reply:
ci/numpy_pandas_requirements.txt  is availble in this PR and ./ci/numpy_pandas_req_pinned.txt is temp file.